### DISCUSSION TITLE
Clear App Data prompt permission error

### DISCUSSION THREAD

**Discussion by maxvoltage** [PlayCover/PlayCover#1150] [Q&A] [answered]
I just upgraded my OS to Sonoma and now PlayCover prompts a permission error when I try to clear app data. Which folder/file it tries to access to clear app data? It seems like it accesses container folder files with no issue.

**Comment by maxvoltage** [comment]
It was actually individual app folders in container folder.

**Comment by ohaiibuzzle** [comment, marked as answer]
On Sonoma you will need to allow PlayCover to have Full Disk Access and App Management permissions
